pymongo/collection.py

```diff
@@ -2584,7 +2584,8 @@ def watch(
 
     def rename(
         self,
-        new_name: str,
+        new_name: Optional[str] = None,
+        to: Optional[str] = None,
```

Comment:
I assume you only added "to" so that the new test would pass. Since we already have "new_name" we don't need a "to" parameter. Let's remove it and instead change the unified test runner to translate to "to" parameter to "new_name". We already do this kind of parameter translation here: https://github.com/mongodb/mongo-python-driver/blob/5578999a90e439fbca06fc0ffc98f4d04e96f7b4/test/utils.py#L1021

Reply:
Oh good, will do